src/app/frontend/common/pipes/coreformatter.ts

```diff
@@ -23,6 +23,7 @@ export class CoreFormatter extends DecimalPipe {
   readonly base = 1000;
   readonly powerSuffixes = ['m', '', 'k', 'M', 'G', 'T'];
 
+  // @ts-ignore
```

Comment:
Can you explain this?

Reply:
it looks like a typescript compiler error

```
Error: src/app/frontend/common/pipes/coreformatter.ts:26:3 - error TS2416: Property 'transform' in type 'CoreFormatter' is not assignable to the same property in base type 'DecimalPipe'.
  Type '(value: number) => string' is not assignable to type '{ (value: string | number, digitsInfo?: string, locale?: string): string; (value: null, digitsInfo?: string, locale?: string): null; (value: string | number, digitsInfo?: string, locale?: string): string; }'.
    Type 'string' is not assignable to type 'null'.

26   transform(value: number): string {
     ~~~~~~~~~
src/app/frontend/common/pipes/memoryformatter.ts:26:3 - error TS2416: Property 'transform' in type 'MemoryFormatter' is not assignable to the same property in base type 'DecimalPipe'.
  Type '(value: number) => string' is not assignable to type '{ (value: string | number, digitsInfo?: string, locale?: string): string; (value: null, digitsInfo?: string, locale?: string): null; (value: string | number, digitsInfo?: string, locale?: string): string; }'.
    Type 'string' is not assignable to type 'null'.

26   transform(value: number): string {
     ~~~~~~~~~
```

Reply:
with `@ts-ignore`, the *tsc* command will ignore this error

Reply:
It has to be fixed then, not ignored

Reply:
must we fix it in this pr?

Reply:
100%, I prefer to keep Angular 10 without any hidden issues than go to the 11 but with workarounds and ignored errors.

Reply:
Have you tested if the pipes works with the introduced annotations and there are no errors in the console?